users/models.py

```diff
@@ -1,3 +1,21 @@
 from django.db import models
 
 # Create your models here.
+class User (models.Model):
```

Comment:
Instead of creating a user model from scratch, it is better to extend the **AbstractUser**. It makes it much easier to implement our authentication system. For more details please refer here: 
https://simpleisbetterthancomplex.com/tutorial/2016/07/22/how-to-extend-django-user-model.html#abstractuser